langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -75,7 +129,13 @@ public isolated function round(decimal x, int fractionDigits = 0) returns decima
 } external;
 
 # IEEE quantize operation.
-
```

Comment:
In this part, seems like `Ballerina Specification` contains more details
```
# Return a decimal with a specified value and exponent.
# Return a decimal value that has the same value (except for rounding) as the first
# argument, and the same exponent as the second argument.
# This is the IEEE quantize operation.
# + x - decimal value to operate on
# + y - decimal value from which to get the quantum
# + return - `x` with the quantum of `y`
public isolated function quantize(decimal x, decimal y) returns decimal = external;
```

Shall we update it in this PR also?, or in a separate PR?
wdyt?

Reply:
Spec and langlib api docs should have a 1 to 1 mapping. You should add these.